tests/lib/fakegpio/fake-gpio.py

```diff
@@ -59,6 +60,15 @@ def dispatch(sel):
     return True
 
 
+def maybe_sd_notify(s: str) -> None:
+    addr = os.getenv('NOTIFY_SOCKET')
+    if not addr:
+        return
+    soc = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+    soc.connect(addr)
+    soc.sendall(s.encode())
+
+
```

Comment:
Nitpick, double newline.

Reply:
I added those because pep8 wants double newlines between functions that are not part of a class.